src/modules/sub-items/components/pagination/pagination.btn.component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PaginationBtnComponent = ({ label, disabled, additionalClasses, onPageChange, page }) => {
+    const onClick = () => {
+        if (!disabled && page) {
+            onPageChange(page);
+        }
+    };
+
+    let className = `page-item ${additionalClasses}`;
+
+    className = disabled ? `${className} disabled` : className;
+
+    return (
+        <li className={className}>
+            <button className="page-link" onClick={onClick}>
```

Comment:
`onClick` -> `onClick` is for me not proper. Because `on click` you do not do `on click` but rather `handle click`

Reply:
I like this proposition. I was curious to know how it was for the rest of the code in `admin-ui-modules`, so I made small search and for `={han` search there are 2 results, and for `={on` there are 25 results, which have been written by me, you and @sunpietro . 😛 

I will gladly change it to `handleClick` because it feels better but it's hard to write code which satisfies others when there are no clear rules written down. I think I will start writing them down. 😄

Reply:
@tischsoic It is a very good idea!

Reply:
There is so many `={on` because of passing prop from parent to the child component. This is ok, the prop can be named `onSomething`, we should avoid naming method `onSomething`. 

When I checked for `={this.on` there are a few matches (mostly from the beginning of v2 where we were in kinda rush).

Reply:
Thank you for reply and clarification @dew326 , now it makes sense to me. :)

Reply:
During implementation of bulk delete/move I introduced `onSelectAll` method: https://github.com/ezsystems/ezplatform-admin-ui-modules/blob/946fdedb0895ffb7ed811adcfb0884bdb64adbbc/src/modules/sub-items/components/table-view/table.view.component.js#L154
I'm wondering, should it be named `selectAll` or `handleSelectAll`? ping @dew326 :)

Reply:
OK, now I saw other comments and I understand it should be renamed to `selectAll`.